components/climate/ir_climate.rst

```diff
@@ -62,8 +62,8 @@ controller unit.
       - platform: coolix       # adjust to match your AC unit!
         name: "Living Room AC"
 
-Configuration variables:
-------------------------
+Configuration Variables
```

Comment:
By undocumented convention this title ends in : (see all other pages) if you want you can make it not be a title. (But keep the :)

It's actually expected the : for automatic parsing of it.

Reply:
Noted! 😄